tests/wamr-test-suites/wasi-test-script/run_wasi_tests.sh

```diff
@@ -15,22 +15,22 @@ readonly IWASM_CMD="${WORK_DIR}/../../../../product-mini/platforms/${PLATFORM}/b
     --allow-resolve=google-public-dns-a.google.com \
     --addr-pool=::1/128,127.0.0.1/32"
 
-readonly IWASM_CMD_STRESS="${IWASM_CMD} --max-threads=8"
```

Comment:
I wonder if there's a way to define this number once, instead of doing it here AND in some of the test files?

Reply:
I don't know any except just setting an env var but I can check 

I don't think test file can access any iwasm settings but I might be wrong